src/config.rs

```diff
@@ -158,40 +158,45 @@ impl TitanCfConfig {
     }
 }
 
-fn get_background_job_limit(
-    default_background_jobs: i32,
-    default_background_flushes: i32,
-    default_sub_compactions: u32,
-    default_background_gc: i32,
-) -> (i32, i32, u32, i32) {
+#[derive(Clone, Copy, Debug)]
+struct BackgroundJobLimits {
```

Comment:
This is much better, thanks! As a rule of thumb I try to avoid having 2 arguments of the same type given to a function.

Reply:
I think arguments are fine. They are named. I'm bother by the unnamed return tuple.

Reply:
The return tuple was of variables (named) at both the function definition return and the function call return. The calling bothered me more because there were unnamed numbers in some spots. They both bother me a lot, thanks again for cleaning it up.